cfn-guard-lambda/README.md

```diff
@@ -19,6 +19,7 @@ The primary interface for building and deploying the tool is the [Makefile](Make
 * If building on a Mac, you'll need [Homebrew](https://brew.sh/).  
 * If building on Ubuntu, you'll need to run `sudo apt-get update; sudo apt install build-essential` if you haven't already
 * If building on CentOS/RHEL you'll need to add the `musl-libc` package repository to your yum config (see https://copr.fedorainfracloud.org/coprs/ngompa/musl-libc/)
+* If leveraging, cloud infrastructure in code and provisioning it through AWS CloudFormation, you'll need to install or update the [AWS CDK CLI] from npm (requires [Node.js ≥ 10.13.0](https://nodejs.org/download/release/latest-v10.x/)): `npm i -g aws-cdk`
```

Comment:
Nitpick: I think the CDK is better seen as an option next to CloudFormation.

I also suggest adding a link to the CDK docs, in case the installation instructions or requirements change

```suggestion
* If provisioning with the Cloud Development Kit (CDK), you'll need to install or update the [AWS CDK Toolkit](https://docs.aws.amazon.com/cdk/latest/guide/cli.html) from npm (requires [Node.js ≥ 10.13.0](https://nodejs.org/download/release/latest-v10.x/)): `npm i -g aws-cdk`
```

Reply:
> I'm not opposed to a CDK deployment option, but I don't think this solves #10 and would still like to see a native CloudFormation option too (possibly. with SAM, which can use the transform inside of the CloudFormation service).
> 
> There is a big difference in how you deploy things with CDK and with CloudFormation, and someone using CloudFormation, or someone new to IaaC might prefer a single template to deploy over the typescript code in the cdk app.
> 
> Edit: This is not meant as a criticism of your code, but I'm trying to highlight the difference between what I saw as a problem when I raised #10 (no CloudFormation template for a CloudFormation-related tool) and what this PR solves (no IaaC option for an IaaC-related tool)

I have included steps to generate the CloudFormation template for the deployment, in the _Readme_s of the associated CDK projects *-lambda. So a native CloudFormation option is still available, if that is preferred.

I do see where you are coming from though. While I prefer using CDK for IaaC and managing my deployments, others may prefer CloudFormation templates. I have, on occasion, used the CDK feature, to emit the CloudFormation template, when one has been required.

As for the SAM option, I did explore that and wasn't able to build the project successfully...YMMV.

Happy to create a issue specifying CDK support, if this PR don't solve #10